scripts/modelbench/inductor_single_run.sh

```diff
@@ -43,24 +43,16 @@ if [[ "$DT" == "amp_fp16" ]]; then
     DT_extra="--amp-dtype float16 "
 fi
 
-Flag_extra=""
 cd /workspace/pytorch/benchmarks/dynamo
 if [[ $BACKEND == "aot_inductor" ]]; then
-    echo "Testing with aot_inductor."
     # Workaround for test with runner.py
-    sed -i '/"inference": {/a \ \ \ \ \ \ \ \ "aot_inductor": "--inference -n50 --export-aot-inductor ",' runner.py
-    echo "Setting freezing for inductor backend by default."
-    export TORCHINDUCTOR_FREEZING=1
-    Flag_extra+="--freezing $(python -c "import runner; print(runner.TABLE['${MODE}']['${BACKEND}'])") "
-elif [[ $BACKEND == "inductor" ]]; then
-    echo "Setting freezing for inductor backend by default."
-    export TORCHINDUCTOR_FREEZING=1
-    Flag_extra+="--freezing $(python -c "import runner; print(runner.TABLE['${MODE}']['${BACKEND}'])") "
-elif [[ $BACKEND == "inductor_max_autotune" ]]; then
-    echo "Setting freezing for inductor with max autotune by default."
-    export TORCHINDUCTOR_FREEZING=1
-    Flag_extra+="--freezing $(python -c "import runner; print(runner.TABLE['${MODE}']['${BACKEND}'])") "
+    sed -i '/"inference": {/a \ \ \ \ \ \ \ \ "aot_inductor": "--inference -n50 --export-aot-inductor ",' runner.py    
 fi
+
+echo "Testing with ${BACKEND}."
+export TORCHINDUCTOR_FREEZING=1
+Backend_extra="--freezing $(python -c "import runner; print(runner.TABLE['${MODE}']['${BACKEND}'])") "
```

Comment:
Why we put the `--freezing` into the `Backend_extra`? if the MODE is training, we don't need it. Using runner.py helps to improve the backend flexibility of this script, but somehow it's not very intuitive. How do you think?

Reply:
Good suggestion. What about now, updated on commit: https://github.com/chuanqi129/inductor-tools/pull/132/commits/e1e0f248993f72684a2b670f1f5d4c9346a00816